lib/datasource/cargo/index.ts

```diff
@@ -1,6 +1,7 @@
 import { logger } from '../../logger';
 import got from '../../util/got';
 import { PkgReleaseConfig, ReleaseResult, Release } from '../common';
+import * as errorTypes from '../../constants/error-messages';
```

Comment:
```suggestion
import { REGISTRY_FAILURE } from '../../constants/error-messages';
```

I would prefer to direct import the specific errors.
@rarkins @JamieMagee What do you think?

Reply:
Yes, agreed. @souravdasslg could you make the changes?

Reply:
Sure.